### ISSUE TITLE
Nvim-treesitter requires Neovim 0.8.3+

### ISSUE THREAD

**Issue by polydevuk** [qvacua/vimr#1009]
NeoVim 0.9.1 via MacPorts on Mac OS Ventura

:checkhealth

nvim-treesitter: require("nvim-treesitter.health").check()
========================================================================
## Installation
  - ERROR: Nvim-treesitter requires Neovim 0.8.3+

**Comment by wookayin** [comment]
Yes, vimr is based on neovim 0.8.2; until #1007 you should pin treesitter at older versions.

**Comment by huyz** [comment]
Which nvim-treesitter should we pin to, so that we don't all have to dig?

**Comment by wookayin** [comment]
I pinned at v0.9.1 and it works fine. See https://github.com/nvim-treesitter/nvim-treesitter/issues/2293

**Comment by georgeharker** [comment]
Please have a try with the newer snapshots which have a newer neovim and can point at external neovim also.

**Comment by huyz** [comment]
> can point at external neovim also. 

I can't see to find documentation on how to point to an external neovim. How can we do this?

**Comment by georgeharker** [comment]
In advanced settings you can paste the path of the binary

**Comment by huyz** [comment]
This ticket can be closed
